### ISSUE TITLE
deprecation warning in tests

### ISSUE THREAD

**Issue by tansaku** [AgileVentures/WebsiteOne#2285]
seeing this in jasmine and rspec tests:

```
/Users/tansaku/.rvm/gems/ruby-2.5.0/gems/business_time-0.7.6/lib/business_time/core_ext/fixnum.rb:6: warning: constant ::Fixnum is deprecated
```

**Comment by NickSchimek** [comment]
The Gem monkey patches into Fixnum which was unified into Integer in Ruby 2.4 which is why we get the deprecation warning. The business_time gem is a dependency of the pivotal-tracker-api gem which hasn't seen any maintenance in the last two years. I've forked the gem to update the dependencies, however prior to updating the dependencies the tests are not passing as it seems one of them is trying to make a call to an API... I've got a branch with the updated dependencies, but would prefer passing tests before being merged (even though our tests `should` catch any issues). Investigating further.

**Comment by NickSchimek** [comment]
Yes, verified that it shouldn't be an issue merging in the branch as the failing pivotal-tracker-api test is trying to make an API call to a project that I don't have access too. Also the deprecated methods in business time are not used in the pivotal-tracker-api which explains why we have not seen any issues when upgrading our Ruby version. It should be straight forward to update business time gem. 
Since it's not an urgent issue for us I went ahead and created an issue and a pull request to update the official pivotal-tracker-api as the owner of the gem seems to maintain it still, but the response may be slow.
I will wait an indeterminate amount of time to see if there is any response from the PR, and then if there is no response after the wait period I will update websiteone to use AV's Forked version.

**Comment by tansaku** [comment]
sounds good - thanks @NickSchimek
